### PR TITLE
Remove remote flag

### PR DESCRIPTION
**Pull request by tobespc**
remove all the remote flag code, fix . up some tests and temporarily disable a few whilst we work out what is failing

### CODE REVIEW

File: src/pfe/file-watcher/server/src/projects/projectUtil.ts [hunk 1]
```diff
@@ -50,13 +50,15 @@ if (process.env.IN_K8) {
...
 const KUBE_NAMESPACE = process.env.KUBE_NAMESPACE || "default";
-const REMOTE_MODE = (process.env.REMOTE_MODE) || false;
+
 
 const containerInfoMap = new Map();
 
 export const containerInfoForceRefreshMap = new Map();
 
-export const LOCAL_WORKSPACE = REMOTE_MODE === "true" ? "/codewind-workspace" : (process.env.NODE_ENV === "test" ? process.env.HOST_WORKSPACE_DIRECTORY : (process.argv[2] ? process.argv[2] : process.env.HOST_WORKSPACE_DIRECTORY));
+
```
Comment (ssh24) [inline src/pfe/file-watcher/server/src/projects/projectUtil.ts]: Can we remove the empty lines?

File: src/pfe/file-watcher/server/src/projects/projectUtil.ts [hunk 2]
```diff
@@ -50,13 +50,15 @@ if (process.env.IN_K8) {
 }
 
 const KUBE_NAMESPACE = process.env.KUBE_NAMESPACE || "default";
-const REMOTE_MODE = (process.env.REMOTE_MODE) || false;
+
```
Comment (ssh24) [inline src/pfe/file-watcher/server/src/projects/projectUtil.ts]: same here

File: src/pfe/iterative-dev/idc-java/IDC/src/org/eclipse/codewind/iterdev/Constants.java [hunk 3]
```diff
@@ -124,8 +124,7 @@
 	public static final String LOCAL_WORKSPACE_ORIGIN = "localWorkspaceOrigin";
 	public static final String LOCAL_WORKSPACE_ORIGIN_KEY = "--localWorkspaceOrigin=";
 
-	public static final String TURBINE_SYNC = "turbineSync";
-	public static final String TURBINE_SYNC_KEY = "--turbineSync=";
+
```
Comment (ssh24) [inline src/pfe/iterative-dev/idc-java/IDC/src/org/eclipse/codewind/iterdev/Constants.java]: same here

File: src/pfe/file-watcher/server/src/projects/projectUtil.ts (hunk 1 above)
Comment (tobespc) [reply src/pfe/file-watcher/server/src/projects/projectUtil.ts]: will remove the lines, all the test pass and manual testing is proving its all ok

**Comment by tobespc** [comment]
@ssh24 thanks for reviewing

**Pull request merged**